### PR TITLE
[docker-platform-monitor]: Add fancontrol

### PR DESCRIPTION
**Pull request by jleveque**
Resolves https://github.com/Azure/sonic-buildimage/issues/705

### CODE REVIEW

**Comment by lguohan** [comment]
@NStetskovych, can you help to test this on your platform?

File: dockers/docker-platform-monitor/supervisord.conf [hunk 1]
```diff
@@ -27,9 +27,17 @@ autorestart=false
 stdout_logfile=syslog
 stderr_logfile=syslog
 
+[program:fancontrol]
+command=/usr/sbin/fancontrol
```
Comment (stcheng) [inline dockers/docker-platform-monitor/supervisord.conf]: why sbin?

Comment (jleveque) [inline dockers/docker-platform-monitor/supervisord.conf]: That's simply where it gets installed via the .deb package.

**Comment by jleveque** [comment]
Retest this please.

**Comment by NStetskovych** [comment]
looks good:
root@sonic:/home/admin# cat /var/log/syslog | grep fan
Jun 23 05:51:03.742334 sonic INFO pmon.sh[1573]: 2017-06-23 05:51:03,741 INFO spawned: 'fancontrol' with pid 19
Jun 23 05:51:03.0 sonic INFO supervisord: fancontrol Loading configuration from /etc/fancontrol ...
Jun 23 05:51:03.0 sonic INFO supervisord: fancontrol 
Jun 23 05:51:03.0 sonic INFO supervisord: fancontrol Common settings:
Jun 23 05:51:03.0 sonic INFO supervisord: fancontrol   INTERVAL=10
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol 
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol Settings for /sys/bus/i2c/devices/2-0066/fan_pwm:
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol   Depends on /sys/bus/i2c/devices/3-004b/hwmon/hwmon[[:print:]]*/temp1_input
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol   Controls /sys/bus/i2c/devices/2-0066/fan1_front_speed_rpm
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol /sys/bus/i2c/devices/2-0066/fan2_front_speed_rpm
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol /sys/bus/i2c/devices/2-0066/fan3_front_speed_rpm
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol /sys/bus/i2c/devices/2-0066/fan4_front_speed_rpm
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol /sys/bus/i2c/devices/2-0066/fan5_front_speed_rpm
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol /sys/bus/i2c/devices/2-0066/fan6_front_speed_rpm
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol   MINTEMP=20
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol   MAXTEMP=60
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol   MINSTART=150
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol   MINSTOP=7
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol   MINPWM=0
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol   MAXPWM=255
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol 
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol Settings for /sys/bus/i2c/devices/2-0066/fan_pwm:
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol   Depends on /sys/bus/i2c/devices/3-004a/hwmon/hwmon[[:print:]]*/temp1_input
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol   Controls /sys/bus/i2c/devices/2-0066/fan1_front_speed_rpm
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol /sys/bus/i2c/devices/2-0066/fan2_front_speed_rpm
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol /sys/bus/i2c/devices/2-0066/fan3_front_speed_rpm
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol /sys/bus/i2c/devices/2-0066/fan4_front_speed_rpm
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol /sys/bus/i2c/devices/2-0066/fan5_front_speed_rpm
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol /sys/bus/i2c/devices/2-0066/fan6_front_speed_rpm
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol   MINTEMP=20
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol   MAXTEMP=60
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol   MINSTART=150
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol   MINSTOP=7
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol   MINPWM=0
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol   MAXPWM=255
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol Enabling PWM on fans...
Jun 23 05:51:04.0 sonic INFO supervisord: fancontrol Starting automatic fan control...
Jun 23 05:51:05.642549 sonic INFO pmon.sh[1573]: 2017-06-23 05:51:05,340 INFO success: fancontrol entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
Jun 23 05:51:05.0 sonic INFO supervisord: start.sh fancontrol: started
Jun 23 05:51:12.0 sonic INFO supervisord 2017-06-23 05:51:03,741 INFO spawned: 'fancontrol' with pid 19
Jun 23 05:51:12.0 sonic INFO supervisord 2017-06-23 05:51:05,340 INFO success: fancontrol entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)

**Comment by jleveque** [comment]
Thanks, @NStetskovych!

**Comment by jleveque** [comment]
Retest this please.

**Pull request merged**